components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
+AML
+AnaLysis
+ASXL
+Azacytidine
+arounds
+Baumer
+Benjamini
+bioconductor
+bioinformatics
+Brems
+CCDL
+CCDL's
+cheatsheet
+cheatsheets
+ChIP
+Cmd
+ColorBrewer
+ComplexHeatmap
+CREB
+Crouser
+cytosolic
+Danio
 dataset
 dataset's
+decitabine
 DESeq
```

Comment:
I notice that most function names are not here, which leads me to think that `DESeq()` will not trigger a spelling error. Are we using `DESeq` on its own in a different way?

Reply:
We use it as in talking about `DESeq` but I think the instances its used it should just be changed to `DESeq2`.